### PR TITLE
[DC-3047] Fix the 'NOT RUN' bug in check_controlled_tier.py

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/controlled_tier_qc/code/controlled_tier_qc.py [hunk 1]
```diff
@@ -88,20 +88,26 @@ def display_check_summary_by_rule(checks_df, to_include):
         nothing_to_report = set(to_include) - set(by_rule['rule'])
 
         by_rule = by_rule.merge(check_description, how='outer', on='rule')
+
+        by_rule['n_row_violation'] = by_rule['n_row_violation'].fillna(
+            0).astype(int)
+
```
Comment (brendagutman) [inline data_steward/analytics/cdr_ops/controlled_tier_qc/code/controlled_tier_qc.py:94]: Moved from below so changes to 'note' would be together. I did the same in the else statement.

**Review by ksdkalluri — CHANGES_REQUESTED** [review]
@brendagutman, changes look good to make this work. Can you also update the analytics/cdr_ops/controlled_tier_qc/Readme.md and add in point that check needs to be added in two places. Thanks!

**Comment by brendagutman** [comment]
> @ksdkalluri  I added an acceptance criteria to the resulting ticket [DC-3125](https://precisionmedicineinitiative.atlassian.net/browse/DC-3125). To add the check to the README and to `to_include`.

**Review by ksdkalluri — COMMENTED** [review]
hey @brendagutman we still need to update the README for this ticket.

**Review by ksdkalluri — APPROVED** [review]

**Pull request merged**